Dockerfile.mysql

```diff
@@ -0,0 +1,12 @@
+FROM mysql:latest
+
+ENV MYSQL_ROOT_PASSWORD=1234
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
```

Comment:
Hardcoding the MySQL user password is a security risk. Consider using environment variables or Docker secrets to manage sensitive information securely.